src/renderers/webgl/WebGLBindingStates.js

```diff
@@ -379,7 +384,7 @@ function WebGLBindingStates( gl, extensions, attributes, capabilities ) {
 
 					// check for integer attributes (WebGL 2 only)
 
-					const integer = ( capabilities.isWebGL2 === true && ( type === gl.INT || type === gl.UNSIGNED_INT || geometryAttribute.gpuType === IntType ) );
+					const integer = ( capabilities.isWebGL2 === true && ( type === gl.BYTE || type === gl.SHORT || type === gl.INT || type === gl.UNSIGNED_BYTE || type === gl.UNSIGNED_SHORT || type === gl.UNSIGNED_INT || geometryAttribute.gpuType === ByteType || geometryAttribute.gpuType === ShortType || geometryAttribute.gpuType === IntType || geometryAttribute.gpuType === UnsignedByteType || geometryAttribute.gpuType === UnsignedShortType || geometryAttribute.gpuType === UnsignedIntType ) );
```

Comment:
Failing examples are: `misc_exporter_gltf` `webgl_animation_multiple` `webgl_animation_skinning_additive_blending` `webgl_animation_skinning_ik` `webgl_animation_skinning_morph` `webgl_buffergeometry_points_interleaved` `webgl_buffergeometry_rawshader` `webgl_buffergeometry_uint`.

They expect float values instead. I'm not sure how `BufferAttribute.gpuType` is interacting here, but I suspect it's not being respected when explicitly defined as float types or unset. These examples use `InterleavedBufferAttribute` and `BufferAttribute` which doesn't have `gpuType` set.

I'll try only checking against `gpuType` so this defaults to float, but it would be good to confirm expected behavior here.